predeploy.py

```diff
@@ -44,8 +44,10 @@
 with open(os.path.join(app_path, "Procfile"), "r") as f:
     contents = f.read().split(" ")
     for item in contents:
-        if "server" in item:
+        if "server" or "SERVER" in item:
```

Comment:
We should just edit the Procfile for that one app, since it's the only app for which this is the case.

Reply:
> We should just edit the Procfile for that one app, since it's the only app for which this is the case.

This works as well. I could drop this edit and edit nlp Procfile.